### DISCUSSION TITLE
Issue number can't be used as a filter/search

### DISCUSSION THREAD

**Discussion by tamalexmak** [community/community#5999] [Projects and Issues] [answered]
Board view: When I try to search for a card with a specific issue number, it doesn't return any result. 
Table view: These is no way to display and search for an issue number. 

Would be nice to have it as a field or at least as a keyword for search.

**Comment by alexcnichols** [comment, marked as answer]
👋🏼 Improvements for displaying the issue number in the project table, and more, are coming soon! Check out this roadmap item for more and subscribe to be notified if/when things change. https://github.com/github/roadmap/issues/283 :octocat: 

Marking this as a duplicate, in favor of this other discussion:

- https://github.com/github/feedback/discussions/5904
